keyboards/etiennecollin/wave/info.json

```diff
@@ -0,0 +1,109 @@
+{
+    "keyboard_name": "Wave",
+    "maintainer": "etiennecollin",
+    "manufacturer": "Etienne Collin",
+    "url": "https://github.com/etiennecollin/wave",
+    "bootloader_instructions": "Press the button marked RESET on the PCB or bridge the pins marked RST and GND on the Elite-C",
+    "tags": ["ortho", "split"],
+
+    "bootloader": "atmel-dfu",
```

Comment:
```suggestion
```

Reply:
Hi, why should this line be removed? This keyboard is compatible with the Elite-C only as it uses a pin that isn't available on the pro-micros.

Reply:
Done